pages/index.tsx

```diff
@@ -5,7 +5,7 @@ import { AddPhoto } from '@/features/addPhoto/addPhoto';
 import { useGetMeQuery } from '@/shared/api/auth.service';
 import { useTranslation } from '@/shared/hooks/useTranslation';
 import { HeadMeta } from '@/widgets/HeadMeta/HeadMeta';
-import { getLayout } from '@/widgets/Layout/Layout';
+import { getWithAuthLayout } from '@/widgets/Layout/withAuthLayout/WithAithLayout';
 import { useRouter } from 'next/navigation';
 
 function Home() {
```

Comment:
When you replace page layouts, don't forget to type them as `NextPageWithLayout`. This type is described in the `pages/_app.tsx` file